breathe/renderer/sphinxrenderer.py

```diff
@@ -191,6 +251,30 @@ class DomainDirectiveFactory:
             'global': (php.PhpGloballevel, 'global'),
```

Comment:
does this work with using the class directly from `php.`?

Reply:
It will not work with scoped rendering, but when Breathe was changed the PHP domain was not yet updated for Sphinx >= 3, so it is still using the old rendering code.

Reply:
I can quickly edit this as well, should I start a new PR or add it here? I wouldn't be able to test it myself but it's pretty straight forward

Reply:
Unfortunately it's not as easy as just editing this part. There are other places in the renderer where the php directives should be directed to the newer code. But if you start another PR with it I'll be happy to help. Though, markstory/sphinxcontrib-phpdomain#30 is probably blocking.

Reply:
ah shame, I thought this would be a quick fix. I don't have any experience with breathe or php, so ill leave it for now and finish this PR first.